web/cypress/integration/route/create-route-with-upstream.spec.js

```diff
@@ -46,32 +47,27 @@ context('Create Route with Upstream', () => {
     cy.contains('Submit').click();
   });
 
-  it('should enter the Route creator', function () {
+  it('should create route with upstream just created', function () {
```

Comment:
put those into smaller test cases because they are clearer, as for login, could we try to put Line33~Line35 under `beforeAll`?

Reply:
> put those into smaller test cases because they are clearer, as for login, could we try to put Line33~Line35 under beforeAll?

Let me have a try~

Reply:
While learning how to use `beforeAll`, I saw the official recommendation best practices of cypress[1], 
> Anti-Pattern: Coupling multiple tests together. 
Best Practice: Tests should always be able to be run independently from one another and still pass.

> Anti-Pattern: Acting like you’re writing unit tests.
Best Practice: Add multiple assertions and don’t worry about it

As There is an interdependence between those small test cases, it would be better to use more `assertions` in a merged test case.

Reference: 
[1] https://docs.cypress.io/guides/references/best-practices.html#Dangling-state-is-your-friend

Reply:
cc @LiteSun @guoqqqi to know